cli/src/utils/command.ts

```diff
@@ -205,4 +205,29 @@ export default abstract class BaseCommand extends Command {
     }
     return project.applicationId;
   }
+
+  protected async getAppId(appName?: string): Promise<{ applicationId: string }> {
+    try {
+      if ( appName ) {
+        const app  = await this.lycanClient.getAppByName(appName);
+        return {
+          applicationId: app.id,
+        };
+        } else {
+            try {
+              const appId = await this.getLocalAppId();
+              return {
+                applicationId: appId,
+              };
+            } catch {
+              throw new CLIError('"appName" flag not provided and could not locate project settings');
+            }
+        }
+    } catch (err) {
+      if (err.name === 'NotFoundError') {
+        throw new CLIError('The current appName is not found');
```

Comment:
`Could not find deployed application named: ${appName}`

Reply:
True.  But "`appName`" is a weird thing in an error message.

Please log what "current" means, e.g. the current working directory.  Also please rephrase the message, we usually avoid articles (`ידוע`) in error messages.  So it would be something like
```suggestion
        throw new CLIError(`No app in current working directory ${path.resolve()}`);
```
(although I'm not even sure what that would say in Windows).

Reply:
Done.